presto-mysql/src/test/java/com/facebook/presto/plugin/mysql/TestMySqlIntegrationSmokeTest.java

```diff
@@ -151,6 +151,30 @@ public void testMySqlTinyint1()
         assertUpdate("DROP TABLE mysql_test_tinyint1");
     }
 
+    @Test
+    public void testCharTrailingWhitespace()
+            throws Exception
+    {
+        execute("CREATE TABLE tpch.char_trailing_whitespace (x char(10))");
+        assertUpdate("INSERT INTO char_trailing_whitespace VALUES ('test')", 1);
+
+        // Test that the predicate doesn't need to be padded.
+        // Comparing to either the stripped or explicit value works.
+        assertQuery("SELECT * FROM char_trailing_whitespace WHERE x = 'test'", "VALUES 'test'");
+        assertQuery("SELECT * FROM char_trailing_whitespace WHERE x = 'test'", "VALUES 'test '");
+        assertQuery("SELECT * FROM char_trailing_whitespace WHERE x = 'test'", "VALUES 'test      '");
+
+        // Add additional trailing whitespace (and padding to full length) to query predicate.
```

Comment:
remove comment, put the following test case(s) right after the previous one. Having no comments will make the code actually more readable, as difference in lines will be visible.